llama.h

```diff
@@ -116,6 +116,9 @@ extern "C" {
     // Returns the number of tokens in the KV cache
     LLAMA_API int llama_get_kv_cache_token_count(struct llama_context * ctx);
 
+    // Sets the current rng seed.
+    LLAMA_API void llama_set_rng_seed(struct llama_context * ctx, uint32_t seed);
```

Comment:
For consistency, `seed` should be the same type as in `llama_context_params`, `int`.

Reply:
It may also be a good idea to allow using a random seed by passing 0, as in `llama_context_params`.